flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1202,15 +1202,18 @@ public void stopCheckpointScheduler() {
 				currentPeriodicTrigger = null;
 			}
 
-			for (PendingCheckpoint p : pendingCheckpoints.values()) {
-				p.abortError(new Exception("Checkpoint Coordinator is suspending."));
-			}
+			abortPendingCheckpoints();
 
-			pendingCheckpoints.clear();
 			numUnsuccessfulCheckpointsTriggers.set(0);
 		}
 	}
 
+	public void cancelPendingCheckpoints() {
+		synchronized (lock) {
+			abortPendingCheckpoints();
```

Comment:
For consistency, I would suggest to decide on one name either cancel or abort and keep across the call chain.